semgrep-core/src/analyzing/AST_to_IL.ml

```diff
@@ -1462,7 +1476,7 @@ and python_with_stmt env manager opt_pat body =
 (* Defs *)
 (*****************************************************************************)
 
-and function_definition env fdef =
+and function_definition_helper env fdef =
```

Comment:
thx for fixing that; I also don't like overwrite previous def with the same name